### PR TITLE
Arrowhead exercise - Iteration 3

### PR DESCRIPTION
**Pull request by locker1776**

### CODE REVIEW

File: arrowhead/arrowhead.rb [hunk 1]
```diff
@@ -15,20 +17,24 @@ class Arrowhead
...
-      raise "Unknown region, please provide a valid region."
-    end
+  def classify(region, shape)
+    region_info = region_lookup(region)
+    arrowhead_info = shape_lookup(region_info, shape)
+    "You have a(n) '#{arrowhead_info}' arrowhead. Probably priceless."
+  end
+
+  def region_lookup(region)
+    return CLASSIFICATIONS[region] if CLASSIFICATIONS.include? region
```
Comment (oharace) [inline arrowhead/arrowhead.rb]: Use parentheses to wrap parameters.

File: arrowhead/arrowhead.rb [hunk 2]
```diff
@@ -15,20 +17,24 @@ class Arrowhead
...
+    raise "Unknown region, please provide a valid region."
+  end
+
+  def shape_lookup(region_info, shape)
+    return region_info[shape] if region_info.include? shape
```
Comment (oharace) [inline arrowhead/arrowhead.rb]: Use parentheses to wrap parameters.

File: arrowhead/arrowhead.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
-class Arrowhead
+module Arrowhead
+  module_function
```
Comment (oharace) [inline arrowhead/arrowhead.rb]: Why this, instead of `self.classify`, etc.?

**Comment by oharace** [comment]
Looks good. :+1:

File: arrowhead/arrowhead.rb (hunk 3 above)
Comment (locker1776) [inline arrowhead/arrowhead.rb]: Some style guides say that if you are not going to instantiate an instance of a class, then you should make it a module instead.

**Comment by paulghaddad** [comment]
@locker1776 Nice work! You're good to move on.

**Pull request closed without merging**